public/form/css/styles.css

```diff
@@ -341,141 +349,119 @@ body {
 }
 
 section {
-  background-color: rgba(23, 64, 132, 0.8);
-  height: calc(100% + 120px);
-  display: flex;
   justify-content: center;
   align-items: center;
   flex-direction: column;
   min-height: 730px;
+  padding: 0 30px;
 }
 
-section.pwd {
-  padding: 0 20%;
+section.pwd,
+section.reg,
+section.ty {
+  padding: 60px 30px 0 30px;
 }
 
 h1 {
   font-weight: bold;
-  font-size: 70px;
-  margin: 60px 0;
-  text-align: center;
+  font-size: 35px;
+  text-align: left;
+  display: block;
+  width: 100%;
 }
 
-form {
-  display: flex;
-  flex-direction: column;
-  width: 450px;
-  height: 100%;
+
+.ty h1 {
+  margin-bottom: 30px;
 }
 
-.reg form {
-  display: block;
-  text-align: center;
+.ty p {
+  font-size: 14px;
+  line-height: 1.5;
 }
 
-.reg form input[type="submit"] {
-  width: 98%;
-  position: relative;
-  left: -1%;
+/*FORM STYLES*/
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
 }
 
 label {
   text-transform: uppercase;
   font-size: 14px;
   height: 30px;
   line-height: 30px;
-  margin: 15px 0 0;
+  margin: 45px 0 0;
   letter-spacing: 1.5px;
   font-family: helvetica, arial, sans-serif;
 }
 
-input, textarea {
-  height: 30px;
+input, textarea, select {
+  -webkit-appearance: none;
+  height: 40px;
   padding: 0 6px;
+  border-radius: 0;
   border: 0;
-  font-size: 11px;
-  font-family: helvetica, arial, sans-serif;
+  font-size: 14px;
+  font-family: 'roboto', Helvetica, Arial, sans-serif;
   flex-shrink: 0;
 }
 
-textarea {
-  padding: 6px;
-  height: 100px;
-}
-
 input:required {
   box-shadow: inset -6px 0px 0px #ef4323;
 }
 
-.tos span input:required {
-  box-shadow: 0px 0px 0px transparent;
+select {
+  cursor: pointer;
+  color: #a9a9a9;
+  background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTMiIGhlaWdodD0iOSIgdmlld0JveD0iMCAwIDEzIDkiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHRpdGxlPlRyaWFuZ2xlIDEgQ29weTwvdGl0bGU+PHBhdGggZD0iTTEyIDFMNi41IDYuNyAxIDEiIHN0cm9rZS13aWR0aD0iMiIgc3Ryb2tlPSIjNTU1IiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz48L3N2Zz4=") center right 20px no-repeat, #fff;
```

Comment:
Shouldn't this be handled by webpack? There's no way of telling what the "source" svg is.